AgentDiscoveries-Frontend/app/src/components/utilities/user-helper.js

```diff
@@ -4,7 +4,8 @@ export function storeUserInfo(userInfo) {
     window.localStorage.setItem('Token', userInfo.token);
     window.localStorage.setItem('UserId', userInfo.userId);
     window.localStorage.setItem('Admin', userInfo.isAdmin ? 'true' : 'false');
-    window.localStorage.setItem('Agent', userInfo.agentId >= 0 ? 'true' : 'false');
+    window.localStorage.setItem('AgentId', userInfo.agentId >= 0 ? 'true' : 'false');
+    window.localStorage.setItem('isAgent', userInfo.agentId == null ? 'false' : 'true');
```

Comment:
This feels slightly duplicated to me - what if instead of storing both, we store the `agentId` directly, and then update the isAgent function to properly deal with the possibility of null?

E.g. set AgentId as you do, but not `isAgent`, and have isAgent be something like...
```
export function isAgent() {
    return window.localStorage.getItem('AgentId') !== null;
}
```